homeassistant/external/vera

```diff
@@ -1 +1 @@
-Subproject commit 30c59781d63322db2160ff00a4b99f16ead40b85
+Subproject commit 68c025f893e50d19508ad6b7d3b12d4753482310
```

Comment:
Yes, we should. Let me take this comment to shamelessly promote [my PR to fix this](https://github.com/jamespcole/home-assistant-vera-api/pull/1). After that PR is done we can add `REQUIREMENTS` to the platforms. [See sensor.mysensors for an example.](https://github.com/balloob/home-assistant/blob/master/homeassistant/components/sensor/mysensors.py#L39-L40)